### PR TITLE
Some block related mappings

### PR DESCRIPTION
**Pull request by DeadlyMC**
- Mapped fields in `Blocks.java`
- Mapped more block classes
- Mapped a few things in `Block.java`
- I might have fixed a few wrong mappings here and there too

### CODE REVIEW

File: mappings/net/minecraft/StairsBlock.mapping [hunk 1]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_16814 net/minecraft/StairsBlock
```
Comment (shedaniel) [inline mappings/net/minecraft/StairsBlock.mapping:1]: needs block package

File: mappings/net/minecraft/ObsidianBlock.mapping [hunk 2]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_16987 net/minecraft/ObsidianBlock
```
Comment (shedaniel) [inline mappings/net/minecraft/ObsidianBlock.mapping:1]: needs block package

File: mappings/net/minecraft/block/AbstractFluidBlock.mapping [hunk 3]
```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2404 net/minecraft/block/AbstractFluidBlock
```
Comment (hYdos) [inline mappings/net/minecraft/block/AbstractFluidBlock.mapping:1]: yarn names dont use abstract from what i remember correct me if im wrong

Comment (DeadlyMC) [reply mappings/net/minecraft/block/AbstractFluidBlock.mapping:1]: They do, u can do a quick github search on the yarn repo, will be presented with several examples.

**Pull request merged**